tutorials/plot_sensors_decoding.py

```diff
@@ -406,7 +406,7 @@
 # ==========
 # .. [1] Jean-Rémi King et al. (2018) "Encoding and Decoding Neuronal Dynamics:
 #        Methodological Framework to Uncover the Algorithms of Cognition",
-#        in press. https://hal.archives-ouvertes.fr/hal-01848442/
```

Comment:
why removing this link?

Reply:
It timed out on CircleCI, and took a very long time to load when I tried, so it seemed unreliable. Want me to put it back?

Reply:
I put it back with a new entry in `linkcheck_ignore`